_pages/en/strategy-build-vs-buy.md

```diff
@@ -28,510 +26,493 @@ th {
-As such, it is advised that when making build decisions, efforts be made for any given contract to be strictly less than $1 million, and any project is expected to have products able to deliver business value in strictly less than six months, ideally within weeks.
-Contracts strictly greater than $10 million should be exception cases for particularly complex systems[^18].
+This Guiding Policy has been prepared by taking into consideration alignment and compliance with existing policy instruments and does not replace them.
+Stakeholders are expected to still comply with existing policy instruments including, but not limited to:
 
-For further discussion, please reference [Annex 1](#annex-1-context), specifically, [GC IT Project Analysis](#gc-it-project-analysis).
+- ESDC Information Management Policy (being drafted)
+- ESDC [Procurement Policies](http://iservice.prv/eng/finance/purchasing/policy.shtml)
+- ESDC Security Policy (being drafted)
+- ESDC [Target Solution Delivery Model](strategy-target-solution-delivery-model.html)
+- TBS [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601)
+- TBS [Directive on Security Management](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32611)
+- GC [Architecture Principles Standards](https://wiki.gccollab.ca/Government_of_Canada_Architectural_Standards)
 
-### Next Steps
+### Governance, Compliance, and Reporting
 
-This strategy promotes increasing the control that the government has regarding the delivery of digital services pertaining to its mandate.
-In order to do so, the ratio of build vs buy decisions is expected to increase in favour of build decisions, as defined in the [Annex 0: Definitions](#annex-0-definitions) and elaborated upon in the [Buy Software Services; Not Products](#2-buy-software-development-services-over-products) section.
-Note that the ultimate goal is to increase agility through reusing existing solutions by leveraging OSS, which offers the opportunity to increase collaboration both between private and public sectors, as well as between government bodies[^19].
+1. IT Solutions are broken down into manageable parts (IT Products)
+2. Each IT Product must be assessed on its own as well as a part of overall the IT Solution.
+3. IT Products may be bought, built, or adopted (e.g., Open Source Software)
+4. IT Solutions are assessed against the IT Solutions risk assessment framework (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)).
+  As such:
+   1. Complex and core to mandate IT Solutions are architected internally to ESDC.
+   2. Non-complex and not core to mandate IT Solutions may be architected and delivered by third party integrators and/or vendors.
```

Comment:
Can complex and core to mandate be delivered by third party integrators and/or vendors?  Its clear that they have to be architected internally to ESDC.  But not so clear about who can deliver those solutions.

Now that I'm looking at it, will it be clear to folks what 'architected internally to ESDC means'? Do we mean by ESDC employees?  Is there a chance folks will confuse 'architected internally to ESDC' to mean on prem?

Reply:
Hmm, will try to clarify, thanks!